cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -572,6 +572,16 @@
       "Description": "Enable EFA on the compute nodes, enable_efa = compute",
       "Type": "String",
       "Default": "NONE"
+    },
+    "MasterCoreCount": {
+      "Description": "Number of cores to be used in MasterServer. Useful to disable hyperthreading",
```

Comment:
"Number of cores to be used in MasterServer. If -1, cores are kept as is."

Reply:
Changed